extension/icu/scripts/makedata.sh

```diff
@@ -0,0 +1,28 @@
+#rm -rf build
+mkdir build
+pushd build
+
+# download ICU 66
+wget -nc https://github.com/unicode-org/icu/archive/refs/tags/release-66-1.zip
+unzip -o release-66-1.zip
+
+# download ICU 71 (replace with latest version)
+wget -nc https://github.com/unicode-org/icu/archive/refs/tags/release-71-1.zip
```

Comment:
Should the version not be parameterized?

Reply:
I was wondering about that but on the flip side its nice to have the correct version checked in. If I parameterised it I'd have to add another script.

Reply:
For the next PR.  Plus we might want to separate the TZ drops from the ICU versions (they do that for Java).